### PR TITLE
Equinox deletes some JRE1.1 profile

### LINKED ISSUE #424: Equinox deletes some JRE1.1 profile
The related issues
- https://github.com/eclipse-equinox/equinox/issues/571
- https://github.com/eclipse-m2e/m2e-core/issues/1735

https://github.com/eclipse-equinox/equinox/issues/571 deletes some JRE1.1 profiles - https://github.com/laeubi/equinox/commit/94661454f2a68450be35490b0f5eda7ba08ea833
These profiles should be deleted from https://github.com/eclipse-jdt/eclipse.jdt.debug/blob/master/org.eclipse.jdt.launching/plugin.xml#L268

### PR DESCRIPTION
**Pull request by snjeza**
## What it does
Fixes #424 

## How to test
See https://github.com/eclipse-m2e/m2e-core/issues/1735

## Author checklist

- [x] I have thoroughly tested my changes
- [x] The change is following the [coding conventions](https://wiki.eclipse.org/Platform/How_to_Contribute#Coding_Conventions)
- [x] I have signed the [Eclipse Contributor Agreement (ECA)](https://www.eclipse.org/legal/ECA.php)

### CODE REVIEW

File: org.eclipse.jdt.launching/plugin.xml [hunk 1]
```diff
@@ -278,10 +278,6 @@
             description="%environment.description.10"
             id="OSGi/Minimum-1.2"
             ruleParticipant="org.eclipse.jdt.internal.launching.environments.DefaultAccessRuleParticipant"/>            
-      <environment
-            description="%environment.description.2"
```
Comment (iloveeclipse) [inline org.eclipse.jdt.launching/plugin.xml:282]: Could you please also delete the description key/value from properties

Comment (snjeza) [reply org.eclipse.jdt.launching/plugin.xml:282]: Fixed.

**Comment by snjeza** [comment]
@iloveeclipse I have updated the PR.

File: org.eclipse.jdt.launching/plugin.xml (hunk 1 above)
Comment (rgrunber) [reply org.eclipse.jdt.launching/plugin.xml:282]: - https://github.com/eclipse-jdt/eclipse.jdt.debug/blob/7c57761f182f30e385b7c980f1f80390d75eadc2/org.eclipse.jdt.launching/schema/executionEnvironments.exsd#L202 .

- https://github.com/eclipse-jdt/eclipse.jdt.debug/blob/7c57761f182f30e385b7c980f1f80390d75eadc2/org.eclipse.jdt.launching/launching/org/eclipse/jdt/internal/launching/environments/ExecutionEnvironmentAnalyzer.java#L67

- https://github.com/eclipse-jdt/eclipse.jdt.debug/blob/7c57761f182f30e385b7c980f1f80390d75eadc2/org.eclipse.jdt.launching/launching/org/eclipse/jdt/internal/launching/environments/ExecutionEnvironment.java#L495

I would have a look if those can be safely removed as well.

Comment (snjeza) [reply org.eclipse.jdt.launching/plugin.xml:282]: I have removed it.

Comment (iloveeclipse) [reply org.eclipse.jdt.launching/plugin.xml:282]: Could you push a PR with this nevertheless, to see which / how many tests will fail?

Comment (snjeza) [reply org.eclipse.jdt.launching/plugin.xml:282]: I have updated the PR. It should work now.

**Comment by snjeza** [comment]
@rgrunber @iloveeclipse Could you, please, review the PR again?

**Review by rgrunber — APPROVED** [review]
LGTM. I tested by adding something like `Arrays.asList(JavaRuntime.getExecutionEnvironmentsManager().getExecutionEnvironments()).stream().anyMatch(ee -> "JRE-1.1".equals(ee.getId()));` and confirm it's `false` with this change.

If @iloveeclipse is fine with it, I'd say we should merge.

**Pull request merged**